README.md

```diff
@@ -90,18 +90,23 @@ component.hbs:
     Create New Organization
   {{/b.header}}
 
-  {{#b.body classNames="no-padding"}}
+  {{#b.body paddingSize='0'}}
```

Comment:
Let's make this an integer rather than a string. It doesn't really matter, btu makes it a little clearer.